ros/src/twist_controller/cfg/DynReconf.cfg

```diff
@@ -8,10 +8,10 @@ PACKAGE = "twist_controller"
 
 gen = ParameterGenerator()
 
-gen.add("dyn_velo_proportional_control", double_t, 0, "Velocity proportional control constant", 0.5, 0, 4)
-gen.add("dyn_accel_proportional_control", double_t, 0, "Acceleration proportional control constant", 1.0, 0, 4.0)
-gen.add("dyn_accel_integral_control", double_t, 0, "Acceleration integral control constant", 0, 0, 1)
+gen.add("dyn_velo_proportional_control", double_t, 0, "Velocity proportional control constant", 0.1, 0, 4)
+gen.add("dyn_velo_integral_control", double_t, 0, "Velocity integral control constant", 0.01, 0, 4.0)
 
-gen.add("dyn_acceleration_cruise_constant", double_t, 0, "Acceleration cruise constant", 0.015, 0, 0.1)
```

Comment:
I've ditched the cruise constant; the integral term in the velocity pid controller now works properly without throttle hunting.